spec/services/articles/feeds/basic_spec.rb

```diff
@@ -30,7 +30,6 @@
     it "returns articles with score above 0 sorted by user preference scores" do
       user.follow(old_story.user)
       old_story_tag = Tag.find_by(name: unique_tag_name)
-      old_story_tag.update(points: 10)
```

Comment:
I tested this with and without the method call.  There are no changes.  It's unnecessary chatter.